internal/service/wafv2/helper.go

```diff
@@ -545,13 +592,39 @@ func expandWafv2Rule(m map[string]interface{}) *wafv2.Rule {
 		return nil
 	}
 
-	return &wafv2.Rule{
+	rule := &wafv2.Rule{
 		Name:             aws.String(m["name"].(string)),
 		Priority:         aws.Int64(int64(m["priority"].(int))),
 		Action:           expandWafv2RuleAction(m["action"].([]interface{})),
 		Statement:        expandWafv2RootStatement(m["statement"].([]interface{})),
 		VisibilityConfig: expandWafv2VisibilityConfig(m["visibility_config"].([]interface{})),
 	}
+
+	if v, ok := m["rule_label"].(*schema.Set); ok {
```

Comment:
Nit: an precautionary empty set check

```suggestion
	if v, ok := m["rule_label"].(*schema.Set); ok  && v.Len() > 0 {
```